### DISCUSSION TITLE
¿How download version 2.5?

### DISCUSSION THREAD

**Discussion by kevinbarja** [Amit0617/tagEveryone#1] [Q&A] [answered]
Hi!

Congrats for your repo. In your readme https://github.com/Amit0617/tagEveryone/wiki/Updates#version-25 you talk about a new version. ¿How download version 2.5?

**Comment by Amit0617** [comment, marked as answer]
It still has to be released I was working on that so I just updated the wiki in advance during the temporary installation on the chromium browser. It will be released through the releases section and more updates would be on the wiki. Since WhatsApp again got an update I would have to update for that also(apart from chromium support).
